.github/actions/build-service/action.yml

```diff
@@ -30,6 +30,11 @@ runs:
       working-directory: ${{ inputs.servicePath }}
       run: 'npm ci --no-fund --no-audit'
 
+    - name: Build
+      shell: bash
+      working-directory: ${{ inputs.servicePath }}
+      run: 'npm run build'
```

Comment:
We could add some caching-steps around this step (so that multiple test-runs can reuse the same data/code);
But lets not introduce that just-yet.

Lets first create something-to-improve-on ;)

Reply:
Even without cache it looks pretty fast :)

![image](https://github.com/global-121/121-platform/assets/1260402/62cefdfe-caed-49eb-858c-492c05d55dfd)